kolibri/core/assets/src/core-actions.js

```diff
@@ -349,6 +443,85 @@ function stopTrackingProgress(store, Kolibri) {
   updateTimeSpent(store, Kolibri, true);
 }
 
+function saveMasteryLog(store, Kolibri) {
+  const masteryLogModel = Kolibri.resources.MasteryLog.getModel(store.core.logging.mastery.id);
+  masteryLogModel.save(_masteryLogModel(store)).then((newMasteryLog) => {
```

Comment:
@indirectlylit Have you guys figured out when are the opportunities to save the masteryLog and attemptLog?